packages/hash/api/src/graphql/resolvers/index.ts

```diff
@@ -99,6 +100,7 @@ export const resolvers = {
     blocks: loggedInAndSignedUpMiddleware(blocksResolver),
     getEntity: loggedInAndSignedUpMiddleware(getEntityResolver),
     getAllLatestEntities: getAllLatestEntitiesResolver,
+    aggregateEntities: loggedInAndSignedUpMiddleware(aggregateEntitiesResolver),
```

Comment:
The resolver above (which this will eventually replace) does not have loggedInAndSignedUpMiddleware. I think someone told me this was to do with viewing pages without an account and such, do we have any information about this decision, perhaps a task to track the security of this?

cc: @CiaranMn @thehabbos007 @benwerner01

Reply:
I'm not sure why `getAllLatestEntities` does not have the middleware on it - it's the `page` query that needs to be ungated to allow for anonymous viewing of a page.

I would not read any security into these middlewares – we know that the app is insecure (made explicit in the second line in the README) and it will not be secure until we introduce a proper authorization system.

Regardless of these middlewares, in the absence of proper authorization gating on the supply of individual entities (not queries), anyone can sign up for an account, construct a graph in which they own an entity connected to someone else's entities of interest, and retrieve anything.

Reply:
![image](https://user-images.githubusercontent.com/6988668/213406564-a0b7558f-6758-4d30-9850-4e6c82f05f6e.png)
`getAllLatestEntities` is called from the login page without a logged in user. I'm sure @benwerner01 knows the reason :)

Reply:
This still applies in the rebased PR, however we now _only_ have `queryEntities` (we do not have competing query endpoints)